File: SpecialEducationPlanning/SpecialEducationPlanning.Business/Model/Project/EducationTypeModel.cs
using Koa.Domain;
using System.Collections;
using System.Collections.Generic;

namespace SpecialEducationPlanning
.Business.Model;

public class HousingTypeModel : BaseModel<int>
{
    public int HousingSpecificationId { get; set; }
    public string Code { get; set; }
    public string Name { get; set; }
    public int? PlanId { get; set; }
    public HousingSpecificationModel HousingSpecificationModel { get; set; }
    public virtual ICollection<PlanModel> Plan { get; set; } = new List<PlanModel>();
}

File: SpecialEducationPlanning/SpecialEducationPlanning.Web/src/app/shared/models/Education-type.model.ts
import { HousingSpecification } from "./housing-specification.model";
import { Plan } from "./plan";

export interface HousingType {
    housingSpecificationId: number;
    code?: string;
    name?:	string;
    planId?: number;
    housingSpecificationModel: HousingSpecification
    plan: Plan;
    id: number;
}